DI-435/README.md

```
@@ -1,43 +1,84 @@
+
+#### no_qc mode
+How the script works in `no_qc` mode:
+1. Finds all DNAnexus projects with suffix `--assay`
+2. Finds all raw VCFs in each of the DNAnexus projects.
+3. Removes all samples which are duplicates.
+4. Creates a final list of all VCFs to merge and writes this to file with `--outfile_prefix`.
+##### Outputs
+- A TSV listing the VCF files to merge (named by `{outfile_prefix}_files_to_merge.txt`)
+- A TSV of all duplicate files which have been removed (`{outfile_prefix}_all_dup_rows.txt`)
 
 ### Bash script to merge VCFs
```

Markdownlint (0.35.0)

Expected: 1; Actual: 0; Below
Headings should be surrounded by blank lines

(MD022, blanks-around-headings)